### DISCUSSION TITLE
Controlling LED strips

### DISCUSSION THREAD

**Discussion by dchbiv** [isaacrlevin/presencelight#958] [Q&A]
My workplace does not allow me to connect any non workplace wifi devices to the network. I am allowed to control a light via USB ie:BusyLight

I would like to 3d print my own enclosure for the status light and illuminate it with the LED's

Has anyone done this successfully?

**Comment by isaacrlevin** [comment]
hey @dcbhiv Does the LocalHostSerial option not work for you?
